geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureTypes.scala

```diff
@@ -64,6 +63,11 @@ object SimpleFeatureTypes {
     val UpdateBackupMetadata  = "schema.update.backup.metadata"
     val UpdateRenameTables    = "schema.update.rename.tables"
 
+    @deprecated("replaced with IndexZ2Shards and IndexZ3Shards")
```

Comment:
Could we leave z.splits as undeprecated, and just make the others take precedence if present? that would simplify config in the general case, and you've already accounted for the others having higher priority.

Reply:
Sure, do you want the same done in the schema builder?